index.json

```
@@ -24,6 +24,16 @@
       "active_record"
     ]
   },
+  "activestorage": {
+    "dependencies": [
+      "rails"
+    ],
+    "requires": [
+      "rails",
```

My previous comment was not correct, this work locally without upstream changes:
```suggestion
    "dependencies": [
      "railties"
    ],
    "requires": [
```
Rails [contains a single file](https://github.com/rails/rails/blob/v7.0.4/rails.gemspec#L21) and dependencies on all gems (including Railties), Railties [has](https://github.com/rails/rails/blob/v7.0.4/railties/railties.gemspec#L20) what we actually need.